docs/_glossary/OS.md

```diff
@@ -0,0 +1,6 @@
+---
+name: OS 
+show-in: [dg]
+---
+
+The OS is the operating system of the computer which manages the software and hardware on the computer.
```

Comment:
The OS isn't "the operating system", "OS" means "Operating System". Please rephrase. Also, might be good to:
* Change the name to `Operating System (OS)`
* Show in both UG and DG

Reply:
Ok sure!